pubspec.lock

```diff
@@ -7,14 +7,14 @@ packages:
       name: _fe_analyzer_shared
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "22.0.0"
+    version: "30.0.0"
```

Comment:
I think this file should be in `ignore` list

Reply:
it seems already in ignore list (`*.lock`) but it has probably been pushed by mistake. I just deleted the file